### PR TITLE
Create beats.1s.lisp

### PR DESCRIPTION
**Pull request by queitsch**

### CODE REVIEW

**Comment by keithamus** [comment]
This is our first clisp extension. We have not yet configured our build system for clisp. @tresni think we should go ahead and merge, or get clisp support then re-look at this?

File: Time/beats.1s.lisp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/local/bin/clisp
+
+; <bitbar.title>Beats Time</bitbar.title>
+; <bitbar.version>v1.0</bitbar.version>
+; <bitbar.author>Jannis Segebrecht</bitbar.author>
+; <bitbar.author.github>queitsch</bitbar.author.github>
+; <bitbar.desc>Displays Swatch .beats time.</bitbar.desc>
+; <bitbar.dependencies>clisp</bitbar.dependencies>
```
Comment (keithamus) [inline Time/beats.1s.lisp]: It looks like an image is missing here. It would be great if you could please add a screenshot of this plugin.

**Comment by tresni** [comment]
I think it would be good to figure out what the testing for LISP is.  There is no "standard" linting tool for it that I can find.  LISP Critic is probably the closest, but (at least for me) not the most obvious thing to setup.  At a minimum we can make sure it runs by doing the following:

```
brew install clisp
clisp Time/beats.1s.lisp
```

No worse then what we are doing for Swift :-)

Extensions supported should be "lisp" and "cl" from what I can find.

File: Time/beats.1s.lisp (hunk 1 above)
Comment (queitsch) [inline Time/beats.1s.lisp]: I've added a preview image.

**Comment by tresni** [comment]
Okay, I sort-of have lisp-critic working...
```
brew install clisp
https://github.com/g000001/lisp-critic.git ~/common-lisp/
wget https://common-lisp.net/project/asdf/asdf.lisp
clisp --silent -x "(load (compile-file \"asdf.lisp\"))(require \"asdf\")(asdf:load-system :lisp-critic)(lisp-critic:critique-file \"$(pwd)/test.lisp\")" | sed -n "/.*-\{70\}/,\$p"
```

If output is greater then 2 lines, you've got warnings/failures.  This also requires stripping the shebang from the lisp file before running it through the linter.

**Comment by tresni** [comment]
Okay, if you `chmod +x` your plugin tests should pass :)  Sorry it took a while. f77eb77 added `clisp` as a basic test of "is this script valid lisp", but that won't allow dependancies.

**Comment by keithamus** [comment]
@queitsch do you think you could fix the build error for this file? Then we can merge. You just need to `chmod +x Time/beats.1s.lisp; git add Time/beats.1s.lisp; git commit -m'make beats.1s.lisp executable'`

**Comment by queitsch** [comment]
Done. Very sorry that took so long, mail got buried.

**Comment by tresni** [comment]
@queitsch looks like you committed the change in version but not the actual change in permissions.

**Comment by matryer** [comment]
@queitsch you're so close :) please finish this off so it can go onto https://getbitbar.com/

**Comment by tresni** [comment]
42c1a71f4a7efe71ea04f2fed1dad28849172890 is the commit for `+x`

**Pull request merged**